steward/src/cork/proposals.rs

```diff
@@ -119,6 +119,28 @@ async fn poll_approved_cork_proposals(state: &mut ProposalThreadState) -> Result
             Ok(r) => r.into_inner().proposal,
             Err(err) => {
                 if err.code() == Code::NotFound {
+                    // look ahead in case id was skipped
+                    let mut increments = 0;
+                    for i in 1..=5 {
+                        let id_to_check = proposal_id + i;
+                        if gov_client
+                            .proposal(tonic::Request::new(QueryProposalRequest {
+                                proposal_id: id_to_check,
+                            }))
+                            .await
+                            .is_err()
```

Comment:
If the query client returns an error for another reason (rather than `Code::NotFound`) then this might not be accurate.